tests/rules/classes/not-allowed/FullyQualifiedSniffImplements.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+// @expectedError Qualifier should be replaced with an import: "implements \IteratorAggregate"
+
+namespace flyeralarm\Test;
+
+class FullyQualifiedSniffExtends implements \IteratorAggregate
```

Comment:
Fix class name in order to prevent duplicated class name.

```suggestion
class FullyQualifiedSniffImplements implements \IteratorAggregate
```